src/KM_MapEditor.pas

```diff
@@ -215,12 +218,14 @@ function TKMMapEditor.HumanCount: Integer;
 end;
 
 
-procedure TKMMapEditor.UpdateField(aInc: Integer; aCheckPrevCell: Boolean);
+//aStageIncrement - stage increment, can be negative

```

Comment:
I am not sure what else can we say about this method argument. It is really looks pretty obvious why it is needed, so thaat is why there was no any comment about it before.
The reason why I added this comment is that I want to emphasise the fact it could be negative (so it will actually work as decrement, not as increment, which is not obvious from its name.
If you have any suggestions about this comment, I can add them with ease :)